doc/src/manual/arrays.md

```diff
@@ -702,18 +702,21 @@ julia> r
   0.0       0.866567
 ```
 
-## Sparse Matrices
+## Sparse Vectors and Matrices
 
-[Sparse matrices](https://en.wikipedia.org/wiki/Sparse_matrix) are matrices that contain enough
-zeros that storing them in a special data structure leads to savings in space and execution time.
-Sparse matrices may be used when operations on the sparse representation of a matrix lead to considerable
-gains in either time or space when compared to performing the same operations on a dense matrix.
+Julia has built-in support for Sparse vectors and
```

Comment:
Yes, sparse matrices was capitalized before so thats why, but I realize now that its because it was the start of the sentence :stuck_out_tongue: